src/main/java/edu/si/trellis/cassandra/CassandraResourceService.java

```diff
@@ -172,15 +167,11 @@ public String generateIdentifier() {
         return execute(immutableDataInsert);
     }
 
-    @Override
-    public Future<Boolean> create(IRI id, Session session, IRI ixnModel, Dataset dataset, IRI container,
-                    Binary binary) {
+    public Future<Boolean> create(IRI id, Session session, IRI ixnModel, Dataset dataset) {
```

Comment:
I'm confused-- are you not writing against the latest Trellis API?

Reply:
I guess that the SNAPSHOT modifier was holding be back at a previous commit. Will bring it up to date.